### ISSUE TITLE
GC # 64 - Rochester Custom MARCXML XSD

### ISSUE THREAD

**Issue by patrickzurek** [cedelis/xcoaitoolkit#119]
JIRA issue created by: rcook
Originally opened: 2011-09-15 01:41 PM

Issue body:

**Comment by patrickzurek** [comment]
JIRA Coment by user: rcook
JIRA Timestamp: 2011-09-15 01:45 PM

Comment body:

I have had this issue in GC for over a year and would like to either bump this to action or remove it.  There are two parts and am seeking input on both.
1.  We are currently using a custom version that allows more leniency.   Should we release the OAI Toolkit with our custom XSD, or with the LC standard and make ours available?
2.  I also noticed that the LOC version was updated in May 2009 to allow more leniency (see below).  I think this change should be made to our custom XSD too.  Right know, I believe we are running with variances from the LC version both because we have not kept up with their changes and we have made our own changes.

May 21, 2009  - Version 1.2 - 
in subfieldcodeDataType  the pattern 
                          "[\da-z!&quot;#$%&amp;'()_+,-./:;&lt;=&gt;?{}_^`~[]\]{1}"
               changed to:  
                         "[\dA-Za-z!&quot;#$%&amp;'()_+,-./:;&lt;=&gt;?{}_^`~[]\]{1}"
    i.e "A-Z" added after "[\d" before "a-z"  to allow upper case.  This change is for consistency with the documentation.
